test/browser.js

```diff
@@ -35,7 +35,7 @@ test.onFailure(window.browserTestDriver_fail);
 window.__VERSION__ = require('../lerna.json').version;
 
 test('Browser tests', t => {
-  require('./modules');
```

Comment:
Why this change? `test/modules.js` are intended to be run on both Node and Browser tests. ... and it seems like a circular import?

Reply:
Er, not sure how I managed to do that - it was not intended